picard/ui/scripteditor.py

```diff
@@ -354,23 +362,141 @@ def eventFilter(self, object, event):
             self.update_examples()
         return False
 
+    def populate_script_selector(self):
+        """Populate the script selection combo box.
+        """
+        self.ui.preset_naming_scripts.blockSignals(True)
+        self.ui.preset_naming_scripts.clear()
+        script_item = FileNamingScript(
+            script=get_config().setting["file_naming_format"],
+            title=_("Current file naming script saved in configuration"),
+            readonly=False,
+            deletable=False,
+            id="current"
+        )
+        self.set_script(script_item.script)
+        self.ui.preset_naming_scripts.addItem(self.SCRIPT_TITLE_SYSTEM % script_item.title, script_item)
+
+        for script_json in self.naming_scripts:
+            script_item = FileNamingScript().create_from_json(script_json)
+            if script_item.get_value('title'):
+                self.ui.preset_naming_scripts.addItem(self.SCRIPT_TITLE_USER % script_item.title, script_item)
```

Comment:
I noticed something:

If I edit the title through Details, it is prefixed with "User: ", but if I edit it through main dialog "Title:" it isn't.

Reply:
It wasn't using the `self.update_combo_box_item()` helper when saving from main editing screen.  Good catch!